pkg/reconciler/eventlistener/eventlistener_test.go

```diff
@@ -1161,7 +1158,7 @@ func TestReconcile(t *testing.T) {
 		endResources: test.Resources{
 			Namespaces:     []*corev1.Namespace{namespaceResource},
 			EventListeners: []*v1beta1.EventListener{elWithStatus},
-			Services:       []*corev1.Service{elServiceWithAnnotation},
+			Services:       []*corev1.Service{elService}, // Annotation is cleared.
```

Comment:
This looks like a behavior change. I think we decided to not clear the annotation due to a user reported issue - they wanted to attach ingress specific annotations or something only to the service without the EL overwriting it.

Reply:
I'll see what I can do 😁